src/dataexplorer/dataexplorer-application/src/components/PatientInfo/PatientInfo.tsx

```diff
@@ -1,44 +1,256 @@
 import React from 'react';
 import useFetch from 'utils/hooks/useFetch';
-import { IPatient, IPatientIdentifier } from 'types/IPatient';
+import { IPatient, IPatientIdentifier, IRecord } from 'types/IPatient';
+import { Row, Col, Card, Table, message, Empty } from 'antd';
 import PatientQuestionnaireResponses from '../PatientQuestionnaireResponses/PatientQuestionnaireResponses';
 
-const PatientInfo = ({ patientID }: any) => {
+interface IInterfaceProps {
```

Comment:
Would like to give this a more specific name, that shows where this interface belongs

```suggestion
interface IPatientInfoProps {
```